docs/03.API-EXAMPLE.md

```diff
@@ -283,6 +284,12 @@ main (int argc, char * argv[])
       len++;
     }
 
+    /* If the command starts with 'quit', break the loop */
+    if (!strncmp (cmd, "quit", strlen ("quit")))
```

Comment:
I think we can use strcmp simply and break only for "quit". Or is there a reason for this prefix check?